modules/300-prometheus/templates/alertmanager/internal/ingress.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if and (.Values.global.modules.publicDomainTemplate) (.Values.prometheus.internal.alertmanagers.internal) }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: alertmanager
```

Comment:
Do we need a separate domain for it? Or is it possible to use something like `grafana.deckhouse.io/alertmanager`?

Reply:
We used grafana..../alertmanager now